### ISSUE TITLE
关于结果

### ISSUE THREAD

**Issue by Junzibuqiaoranyushen** [ProHiryu/bert-chinese-ner#71]
您好，我想问一下怎么进行查看测试结果，即您的test.png截图中展示的测试结果，以及eval_f = 0.9662649
eval_precision = 0.9668882
eval_recall = 0.9656949
global_step = 135181
loss = 40.160034
这些数据应当在哪看哇，第一次用tensorflow进行train，还望大佬解惑
